scripts/write_long_audio_manifest.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import glob
+import json
+import logging
+import os
+from argparse import ArgumentParser
+from itertools import repeat
+from multiprocessing import Pool
+
+import librosa
+
+
+"""
+This script is designed for inference of frame level Voice Activity Detection (VAD) 
+
+This script serves three goals:
+    (1) Write audio files to manifest
+    (2) Split audio file for avoiding CUDA memory issue
```

Comment:
This is nice. But is there any script to combine back audio paths in the manifest file after performing vad to the same single large audio file?

Reply:
What do you mean "combine back audio paths in the manifest file"? In vad_infer.py, we write each large audio in one txt file.

Reply:
Are you chopping audio file or updating offset?

Reply:
updating offset and duratio